scss/_settings_colors.scss

```diff
@@ -45,11 +45,11 @@ $colors--light-theme--text-muted: #666 !default;
 $colors--light-theme--text-default: #111 !default;
 
 //text-hover: minimum contrast on primary that satisfies contrast checker AA
+$colors--dark-theme--text-default: hsl(0, 0%, 100%) !default;
+$colors--dark-theme--text-disabled: rgba($colors--dark-theme--text-default, 0.55) !default;
+$colors--dark-theme--text-muted: rgba($colors--dark-theme--text-default, 0.55) !default;
+$colors--dark-theme--text-hover: hsl(0, 0%, 56%) !default;
 $colors--dark-theme--background: hsl(0, 0%, 15%) !default;
-$colors--dark-theme--background-highlighted: hsl(0, 0%, 20%) !default;
-$colors--dark-theme--border-default: hsl(0, 0%, 27%) !default;
+$colors--dark-theme--background-highlighted: rgba($colors--dark-theme--text-default, 0.05) !default;
```

Comment:
@lyubomir-popov  It feels like 5% is a bit too subtle, the background/borders are barely visible.

Reply:
it is what is used in jaas.ai, and they are not essential as the white-space and bar reinforce the meaning of the line / active element. For example, vscode only has a bar and not a background:

![image](https://user-images.githubusercontent.com/2741678/77553572-e2663200-6eac-11ea-99ba-808982377b09.png)

So I thikn it is acceptableto keep them subtle.